mteb/tasks/Retrieval/eng/FeedbackQARetrieval.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+class FeedbackQARetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="FeedbackQARetrieval",
+        description="Using Interactive Feedback to Improve the Accuracy and Explainability of Question Answering Systems Post-Deployment",
+        reference="https://arxiv.org/abs/2204.03025",
+        dataset={
+            "path": "lt2c/fqa",
+            "revision": "1ee1cd0",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="precision_at_1",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples=None,
+        avg_character_length=None,
```

Comment:
Would be good to add n_samples and avg_character_length. they can be computed using this script: https://github.com/McGill-NLP/statcan-dialogue-dataset/blob/ef89f82c66fab1f4ee810dd8c4e0502e6f0e08c5/retrieval_task/analyze.py

Reply:
re-opening this as it needs to be added

Reply:
@cslizc fyi

Reply:
added. thanks!